pages/docs/project-management/drafts.mdx

```diff
@@ -0,0 +1,48 @@
+---
+description: Hashboard drafts allow you to stage changes to your project safely and apply them in a single operation when you're ready. 
+---
+
+# Drafts
+
+Drafts are a way to stage changes to your Hashboard project safely. 
+Drafted resources can be created, updated, and deleted, and can be applied to your project in a single operation. 
+
+## Creating a draft
+Drafts live in the [Drafts page](https://hashboard.com/app/p/drafts) within a project. 
+
+To create a draft, click the `New draft` button. You'll be prompted for a name and description, which can be changed later.
+
+Once you've created a draft, you can enter the draft by clicking on the `Start drafting` button. 
+
+While in a draft, you should see an orange `Drafting` button at the top of the page to indicate that you're in a draft. 
+This means you're safe to interact with the project and make changes to resources, knowing that they will not be applied to your project until you're ready.
+
+To exit the draft, click the `X` button on the orange `Drafting` button at the top of the page, or navigate back to the Drafts page.
+
+### Permissions
+
+There are no special permissions required for users to create, view, or apply drafts. 
+However, users have the same permissions within draft mode as they do when interacting with other project resources. 
+
+For example, if a user does not have permission to create a model, they will still not be able to create a model in a draft.
+
+## Validating drafts
+
+When developing a draft, it's possible that the draft may enter an invalid state. 
+
+Similar to a merge conflict, this can happen if changes are made to the project that are incompatible with the changes in the draft.
```

Comment:
Maybe say "... to a merge conflict in git..." people that aren't aware of git at least have some context clues to search this out on their own.